packages/core-graphql/lib/defs/inputs.js

```diff
@@ -12,7 +12,6 @@ module.exports = `
   scalar JSON
   scalar Limit
   scalar Offset
-  scalar Address
```

Comment:
Address is the correct type. You can find it in `core-graphql/lib/resolvers/index.js`.